test/test_mail_helpers.py

```diff
@@ -284,6 +318,239 @@ def test_multiple_emails_to_multiple_recipients(self):
             }''')
         )
 
+    def test_single_email_with_all_three_email_contents_to_single_recipient(self):
+        from sendgrid.helpers.mail import (Mail, From, To, Subject,
+                                           PlainTextContent, HtmlContent, AmpHtmlContent)
+        self.maxDiff = None
+        message = Mail(
+            from_email=From('test+from@example.com', 'Example From Name'),
+            to_emails=To('test+to@example.com', 'Example To Name'),
+            subject=Subject('Sending with SendGrid is Fun'),
+            plain_text_content=PlainTextContent(
+                'and easy to do anywhere, even with Python'),
+            amp_html_content=AmpHtmlContent(amp_html_content),
+            html_content=HtmlContent(
+                '<strong>and easy to do anywhere, even with Python</strong>')
+        )
+        
+        self.assertEqual(
+            message.get(),
+            json.loads(json.dumps(response_content_with_all_three_mime_contents))
+        )
+
+    def test_single_email_with_amp_and_html_contents_to_single_recipient(self):
+        from sendgrid.helpers.mail import (Mail, From, To, Subject,
+                                           PlainTextContent, HtmlContent, AmpHtmlContent)
+        self.maxDiff = None
+        message = Mail(
+            from_email=From('test+from@example.com', 'Example From Name'),
+            to_emails=To('test+to@example.com', 'Example To Name'),
+            subject=Subject('Sending with SendGrid is Fun'),
+            amp_html_content=AmpHtmlContent(amp_html_content),
+            html_content=HtmlContent(
+                '<strong>and easy to do anywhere, even with Python</strong>')
+        )
+
+        response_content = {
+            "content": [
+                {
+                    "type": "text/x-amp-html",
+                    "value": amp_html_content
+                },
+                {
+                    "type": "text/html",
+                    "value": "<strong>and easy to do anywhere, even with Python</strong>"
+                }
+            ],
+            "from": {
+                "email": "test+from@example.com",
+                "name": "Example From Name"
+            },
+            "personalizations": [
+                {
+                    "to": [
+                        {
+                            "email": "test+to@example.com",
+                            "name": "Example To Name"
+                        }
+                    ]
+                }
+            ],
+            "subject": "Sending with SendGrid is Fun"
+        }
+        self.assertEqual(
+            message.get(),
+            json.loads(json.dumps(response_content))
```

Comment:
For parity with the other tests in this file instead of using `json.dumps` to convert `response_content` use a raw string as seen in this [test](https://github.com/sendgrid/sendgrid-python/blob/HEAD/test/test_mail_helpers.py#L77).